### ISSUE TITLE
Blog: add share buttons

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#3157]
Build on the Blog MVP work to fully flesh out capabilities.

- [x] Design has been finalized
- [x] Link to original design ticket: #2911 (original design ticket)
- [x] Link to related ticket: #3029 (MVP ticket) & #3086 3030 (post MVP ticket)
- [x] Link to Redpen: https://redpen.io/p/bha12cf7387de4f7e7
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included
- [x] Hover state for elements are designed

## Requirements Checklist:
- [ ] Sticky Side Share buttons (disappears on mobile)
- [ ] Share buttons at the bottom of the blog post

## Desktop (with hover states):
![Post-MVP-hover-states@2x](https://user-images.githubusercontent.com/20977376/56389974-d97ad400-61f8-11e9-93f0-f3e6c8477616.jpg)

**Comment by mmmavis** [comment]
Do we plan to turn these buttons into two new "button group" styles or these are more one-off things?

**Comment by kristinashu** [comment]
We're exploring that question more here https://github.com/MozillaFoundation/Design/issues/339 but I can see use using these buttons elsewhere (for example on our campaign pages).

**Comment by mmmavis** [comment]
> We're exploring that question more here MozillaFoundation/Design#339 but I can see use using these buttons elsewhere (for example on our campaign pages).

kk I'll treat them as custom style for now until we've decided to move them to the official style guide.

@kristinashu how should each button function? I assume similar to what we have on PNI?

<img width="944" alt="image" src="https://user-images.githubusercontent.com/2896608/57812594-5a5abc00-7722-11e9-9e03-d7678088f767.png">

If so do we have default text to share out?

**Comment by kristinashu** [comment]
Yes same functionality please. Copy can just be:
> [Title of blog post] by @mozilla [url to blog post]

**Comment by mmmavis** [comment]
@kristinashu should we make sure these icons are from set Sabrina designed? I know Twitter icon on Foundation site has been updated but the other three icons still need to be replaced. If you think this is a good idea can you include link to the icon folder? I can generate white icon version myself by editing the source code of the icon SVG myself.

**Comment by kristinashu** [comment]
Let's for sure use them! @beccaklam is looking at this right now in the button audit. Becca, do you think the new icons at max 16px will work here too?

**Comment by mmmavis** [comment]
While we are on this... do we plan to update the pill buttons on PNI to the rectangular ones we are adding to the blog page?

<img width="819" alt="image" src="https://user-images.githubusercontent.com/2896608/58983139-f9eff500-878a-11e9-8815-37304e4b92de.png">

**Comment by beccaklam** [comment]
Sorry for the late reply on this, I was testing out the icons yesterday but got sidetracked by other work. I actually think 24px will work better for the circular buttons, as the 16px looks too small. As for the PNI share buttons, I don't feel a need to change them. I see PNI as more of a unique instance/microsite than an extension of the Foundation site. But others feel free to weigh in.

**Comment by beccaklam** [comment]
> @kristinashu should we make sure these icons are from set Sabrina designed? I know Twitter icon on Foundation site has been updated but the other three icons still need to be replaced. If you think this is a good idea can you include link to the icon folder? I can generate white icon version myself by editing the source code of the icon SVG.

Yes, I'll get you these icons @mmmavis!

**Comment by beccaklam** [comment]
So, fidgeted with the icons again and 16px does work after all ... Here are the icons @mmmavis:
[share-buttons.zip](https://github.com/mozilla/foundation.mozilla.org/files/3262859/share-buttons.zip) I also updated the redpen with the new icons.

**Comment by beccaklam** [comment]
Ah, hold off on the facebook icon @mmmavis, I need to get you the official one instead. Will update my tomorrow morning!

**Comment by mmmavis** [comment]
@beccaklam No rush! I have to shift my focus to get the join us modal on Pulse to work first. 😉

**Comment by mmmavis** [comment]
Blocked on 
#3322 (Implement Grouped Button 'Style')
#3323 (Implement Social Buttons)
